rewrite-maven/src/main/java/org/openrewrite/maven/ChangePluginGroupIdAndArtifactId.java

```diff
@@ -56,6 +56,18 @@ public class ChangePluginGroupIdAndArtifactId extends Recipe {
             example = "my-new-maven-plugin",
             required = false)
     @Nullable
+    String newArtifactId;
+
+    /**
+     * Mistakenly introduced, we restored newArtifactId but let's not break recipes abruptly.
+     */
+    @Option(displayName = "New artifact ID",
+        description = "The new artifact ID to use. Defaults to the existing artifact ID. This property is deprecated, use newArtifactId instead.",
+        example = "my-new-maven-plugin",
+        required = false)
+    @Nullable
+    @Deprecated
+    @SuppressWarnings("DeprecatedIsStillUsed")
     String newArtifact;
```

Comment:
Thanks for taking the extra effort to remain backwards compatible; I ran a quick search to see if anything used the constructor, but thankfully that does not seem to be the case. https://app.moderne.io/results/MSJWn0HY8
Via a run of `org.openrewrite.java.search.FindMethods` with methodPattern `*..ChangePluginGroupIdAndArtifactId <constructor>(..)` against the OpenRewrite GitHub org.